src/components/DatePicker/OverlayComponent/components/MonthWrapper/MonthWrapper.style.ts

```diff
@@ -0,0 +1,54 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { Theme } from 'theme';
+
+export const monthWrapperStyle = () => (theme: Theme): SerializedStyles => css`
+  display: flex;
+  margin: 0 10px;
+  flex-direction: column;
+`;
+
+export const monthHeaderWrapperStyle = () => (theme: Theme): SerializedStyles => css`
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  position: relative;
+`;
+
+export const monthHeaderNavigationIconWrapperStyle = ({
+  position = 'left',
+}: {
+  position: 'left' | 'right';
+}) => (theme: Theme): SerializedStyles => css`
+  cursor: pointer;
+  margin: auto 5px;
```

Comment:
same here with 5

Reply:
it doesnt have to be out of the system in this one as it slightly moves the icon a bit inside